# topic to which the server sends info about this resource
string status_topic

# Temoto namespace from where this query was sent
string temoto_namespace

#string test
